pegasus/routes/hoc_routes.rb

```diff
@@ -45,13 +45,13 @@
 # reenable this in a scalable way or remove it entirely.
 get '/api/hour/begin_learn/:code' do |code|
   only_for ['code.org', 'csedweek.org', partner_sites].flatten
-  pass unless tutorial = DB[:tutorials].where(code: code).first
+  pass unless tutorial = Tutorials.new(:tutorials).find_with_code(code)
```

Comment:
Would there be any performance benefit to instantiating Tutorials once instead of for each of these requests?  I *think* we're okay here, just want to make sure there are no surprises.

Reply:
The comments on this method indicate that it is not used due to performance issues.  This new logic (using the Tutorials class so that we can continue using the old Google Sheet's column names) like has the same scalability problem - 1 db query per http request.  The overhead of instantiate the class probably does not make the performance worse  I did update it just now to find the matching tutorial in the database instead of iterating through the array ResultSet in memory.